Configurations/unix-Makefile.tmpl

```diff
@@ -679,9 +679,12 @@ tags TAGS: FORCE
 
 # Release targets (note: only available on Unix) #####################
 
+# If your tar command doesn't support --owner and --group, make sure to
+# use one that does, for example GNU tar
 TAR_COMMAND=$(TAR) $(TARFLAGS) --owner 0 --group 0 -cvf -
```

Comment:
Nevertheless, if users *are* using the `tar` (or `dist`) target, it would be unwise to make sudden variable changes now.  Chances are that there are scripts that depend on them (such as our own `.travis-create-release.sh`).